packages/tools/devtools/devtools-view/src/components/data-visualization/FluidHandleView.tsx

```diff
@@ -87,5 +88,27 @@ export function FluidHandleView(props: FluidHandleViewProps): React.ReactElement
 		return <TreeItem header={header} />;
 	}
 
-	return <TreeDataView containerKey={containerKey} label={label} node={visualTree} />;
+	//Button to inlcude in header to copy fluid handle
+	const button = (
```

Comment:
I mentioned this in our call, but I think we should hold off on adding the handle-copying until we have the ability to edit values to insert handle IDs. Until then, I think it will just likely confuse users. Waiting will also let us settle on a UI design for the editing buttons and such before adding the copy button.

Reply:
I should just remove it rather than commenting it, right? Also another option to keep most of my changes in my branch until its ready to merge completely. Would that make sense to do? Or should I just make these smaller changes anyways?

Reply:
Yeah, we should remove it altogether. 

As for how to manage your changes - I'm a big advocate for small, scoped PRs. So, I think merging in small changes when we identify that they are ready is the way to go. Merging this PR (with just the tree visuals changes) is a great example IMO.